app/_hub/kong-inc/rate-limiting/index.md

```diff
@@ -3,11 +3,11 @@ name: Rate Limiting
 publisher: Kong Inc.
 redirect_from:
   - /enterprise/0.35-x/rate-limiting/
-version: 2.1.0
+version: 2.2.0
 
 desc: Rate-limit how many HTTP requests a developer can make
 description: |
-  Rate limit how many HTTP requests a developer can make in a given period of seconds, minutes, hours, days, months or years. If the underlying Service/Route (or deprecated API entity) has no authentication layer, the **Client IP** address will be used, otherwise the Consumer will be used if an authentication plugin has been configured.
+  Rate limit how many HTTP requests a developer can make in a given period of seconds, minutes, hours, days, months, or years. If the underlying Service/Route (or deprecated API entity) has no authentication layer, the **Client IP** address will be used, otherwise the Consumer will be used if an authentication plugin has been configured.
```

Comment:
unsure: requests a developer can make? maybe client application, or "...requests can be made..." (yeah, passive, but less specific about request origin)

praise: I like the Oxford comma!  (twice in the same paragraph!)

Reply:
Good point, rephrased it